scripts/node/backup.config.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Similarly, the options for where to backup to should be provided as arguments or by the environment that the image is running in (via env variables in kubernetes or the equivalent in docker compose).

Reply:
With that note, should I remove the default CMD in the current `DockerfileNode`? Seems like it would be better that way if these will ultimately just be used in a k8/compose cluster and not as a one off.